tests/test-desktop-files.lua

```diff
@@ -0,0 +1,34 @@
+local runner = require("_runner")
```

Comment:
I don't think this test has to be an integration one. Given you control the files, it could be a `spec/` one. They are much faster.